src/js/libcs/Operators.js

```diff
@@ -3592,14 +3592,98 @@ evaluator.halfplane$2 = function(args, modifs) {
 //   Geometric elements      //
 ///////////////////////////////
 
-evaluator.allpoints$0 = function(args, modifs) {
+
+// helper for all*(<geoobject>)
+eval_helper.allhelper1arg = function(args, kind, restrict) {
+    if (args.length > 1) return nada;
```

Comment:
Can't happen: you are invoking this from a `all*$1` method so the compiler guarantees that there will be exactly one argument. Unless someone introduced a broken code-internal call, but in that case anything at all may happen.